makederiv

```diff
@@ -92,6 +93,9 @@ elif [[ "${OUTPUT_TYPE}" == "mp3" ]] ; then
 elif [[ "${OUTPUT_TYPE}" == "podcast" ]] ; then
     EXTENSION="mov"
     RELATIVEPATH="access/podcast"
+elif [[ "${OUTPUT_TYPE}" == "waveform" ]] ; then
+    EXTENSION="png"
+    RELATIVEPATH="depictions"
```

Comment:
hmmm, i think this would go to a different place than `makewaveform`. This would go to `objects/depictions` whereas it used to go to `metadata/depictions`.

Reply:
does 332e1ff look ok?